pom.xml

```diff
@@ -55,7 +55,15 @@
 				<artifactId>maven-war-plugin</artifactId>
 				<version>3.3.2</version>
 			</plugin>
-		</plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>15</source>
+                    <target>15</target>
+                </configuration>
+            </plugin>
+        </plugins>
```

Comment:
Java verzió beállítása. Már fentebb beállítottuk ezt a `<propreties>` tagben, viszont én dolgoztam más fejlesztőkörnyezettel és oda kellett valamiért. Szóval ez a változtatás is ignorálható.